administrator/components/com_templates/resources/styles/components/_app.scss

```diff
@@ -0,0 +1,67 @@
+// App.vue
+
+#pageBuilder {
+  border: 0.1em solid;
+  min-height: 50em;
+}
+
+.settings {
+
+  .btn-group {
+    margin: 1rem 0;
+
+    .btn {
+      margin-right: 1rem;
+    }
+  }
+}
+
+.draggable {
+  background-color: $list-bg-color;
+  margin-top: 1rem;
+
+  .list-group-item,
+  .list-group-item:last-child {
+    margin: 1rem .5rem;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $list-bg-color-hover;
+      cursor: grab
+    }
+  }
+
+  .grid-row {
+    margin-left: 15px;
+  }
+
+  .close {
+    float: right;
+    margin: 0.2em 0.5em 0 0.2em;
+    padding: 0 0.2em 0 0.2em;
+    cursor: pointer;
+  }
+
+  button[type=button].list-group-item {
+        cursor: pointer;
+        &:hover {
+            cursor: pointer;
+        }
+    }
+}
+
+#newgrid .modal-body img {
```

Comment:
class

Reply:
To fire up a modal, id is required to link the button to the modal. Anyways, I am working to adopt the modal component system as in com_media.